viewer/packages/data-providers/src/image-360-data-providers/descriptor-providers/datamodels/system-space/Cdf360DataModelsDescriptorProvider.ts

```diff
@@ -24,6 +24,7 @@ import partition from 'lodash/partition';
  * An identifier uniquely determining an instance of a Cognite Data Model
  */
 export type Image360DataModelIdentifier = {
+  source?: 'dm' | 'cdm'; // Optional to not break the existing API. The default value is 'dm'.
   space: string;
```

Comment:
I think we should remove comment about breaking API - but keep the part about default value. I also think this should be written as TSDoc, so e.g.
```
/**
 * Source of the data model, 'dm' for the legacy data model, and 'cdm' for the CoreDM
 */
```

Maybe we should choose something else than `dm` here also :thinking: Like `legacy`, or something?

Reply:
Agreed, fixed in [b24ff79](https://github.com/cognitedata/reveal/pull/4894/commits/b24ff79808b01684a7688d8fc7e2782ba0bb5843)